shower_geometry.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+
+def line_point_distance(xp1, yp1, zp1, cx, cy, cz, x, y, z):
```

Comment:
it might be worth to use something like 1D-np.arrays here, I guess. So that 
 * `p1` - point on line
 * `c` - direction of line
 * `p` - point in space

Then the code would reduce to:
```python
a = np.cross((p - p1), c)
return np.sqrt(np.sum(a**2) / np.sum(c**2))
```

Reply:
OK, thanks, it looks way more better :-)